cmd/skopeo/inspect.go

```diff
@@ -22,19 +38,39 @@ var inspectCmd = cli.Command{
 		if err != nil {
 			logrus.Fatal(err)
 		}
+		rawManifest, err := img.Manifest()
+		if err != nil {
+			logrus.Fatal(err)
+		}
 		if c.Bool("raw") {
-			b, err := img.Manifest()
-			if err != nil {
-				logrus.Fatal(err)
-			}
-			fmt.Println(string(b))
+			fmt.Println(string(rawManifest))
 			return
 		}
 		imgInspect, err := img.Inspect()
 		if err != nil {
 			logrus.Fatal(err)
 		}
-		out, err := json.MarshalIndent(imgInspect, "", "    ")
+		manifestDigest, err := dockerutils.ManifestDigest(rawManifest)
+		if err != nil {
+			logrus.Fatalf("Error computing manifest digest: %s", err.Error())
+		}
+		repoTags, err := img.GetRepositoryTags()
```

Comment:
I'm fine doing it later (hopefully we would freeze the API in the coming weeks so we are still more time to think more about this :) 

We can leave a comment though just in case we forgot about doing this larer